gemstore/index.js

```diff
@@ -1,9 +1,13 @@
 (function() {
-  var app = angular.module('gemStore', []);
+  var app = angular.module('gemStore', ['store-directives']);
 
-  app.controller('StoreController', function(){
-    this.products = gems;
-  });
+  app.controller('StoreController', ['$http', function($http){
+    var store =  this;
+    store.products = [];
+    $http.get('/store-products.json').success(function(data) {
+      store.products = data;
```

Comment:
@Manfred-Diaz This assets seems to be required by it is not part of the repository, please include it